README.md

```diff
@@ -133,3 +134,25 @@ go get -t ./...
 # Run tests
 ginkgo
 ```
+
+### Uploading Binaries
+0. Install [goxc](go get github.com/laher/goxc)
+
+  ```bash
+  go get github.com/laher/goxc
+  ```
+
+0. Set version number in `.goxc.json` file
```

Comment:
Move this single point under the `Compiling and Uploading Binaries` sub header below